expirationd.lua

```diff
@@ -48,6 +48,8 @@ local constants = {
     default_on_full_scan = function(...) end,
     -- default function for start_key
     start_key = function(...) return nil end,
+    -- default function for process_while
+    process_while = function(...) return true end,
```

Comment:
The same as in https://github.com/tarantool/expirationd/pull/52/commits/ee67a1a778f012776d3da33d759a6b1888dbf0d1#r656445444